lib/pf/Switch/HP/Procurve_2920.pm

```diff
@@ -0,0 +1,145 @@
+package pf::Switch::HP::Procurve_2920;
+
+=head1 NAME
+
+pf::Switch::HP::Procurve_2920 - Object oriented module to access SNMP enabled HP Procurve 2920 switches
+
+=head1 SYNOPSIS
+
+The pf::Switch::HP::Procurve_2920 module implements an object 
+oriented interface to access SNMP enabled HP Procurve 2920 switches.
+
+=head1 BUGS AND LIMITATIONS
+
+VoIP not tested using MAC Authentication/802.1X
+
+=cut
+use strict;
+use warnings;
+use Log::Log4perl;
+use Net::SNMP;
+use base ('pf::Switch::HP');
+
+sub description { 'HP ProCurve 2920 Series' }
+
+# importing switch constants
+use pf::Switch::constants;
+use pf::util;
+use pf::config;
+
+# CAPABILITIES
+# access technology supported
+sub supportsWiredMacAuth { return $TRUE; }
+sub supportsWiredDot1x { return $TRUE; }
+sub supportsLldp { return $TRUE; }
+sub supportsRadiusVoip { return $TRUE; }
+# inline capabilities
+sub inlineCapabilities { return ($MAC,$PORT); }
+
+=item getVoipVSA
```

Comment:
Please leave an empty line between those lines : 
Ex:
=item

Your description,....

=cut

You'll need to do this for the other headers too